actors.ts

```diff
@@ -114,6 +130,9 @@ module J2ME {
       this.name = opts.name;
       this.signature = opts.signature;
       this.classInfo = opts.classInfo;
+      if (!(this.classInfo instanceof ClassInfo)) {
+        debugger;
```

Comment:
These and the other instances of the *debugger* statement should be assertions, *console.error* calls, or the like.